pkg/addon-operator/operator.go

```diff
@@ -1481,6 +1507,7 @@ func (op *AddonOperator) HandleModuleRun(t sh_task.Task, labels map[string]strin
 				logEntry.Debugf("ModuleRun '%s' phase", baseModule.GetPhase())
 
 				treg := trace.StartRegion(context.Background(), "ModuleRun-OnStartup")
+				defer treg.End()
```

Comment:
not sure about it, this way it would trace everything from OnStartup stage till the end